readme.md

```diff
@@ -0,0 +1,38 @@
+# UEFI NVMe-oF Boot Implementation
+## Introduction
+This UEFI Nvme-oF Boot Implementation enables a Boot-from-SAN prototype that produces and consumes [ACPI](https://uefi.org/specifications) XSDT [NBFT](https://uefi.org/specs/ACPI/6.5/05_ACPI_Software_Programming_Model.html?highlight=nbft#description-header-signatures-for-tables-reserved-by-acpi) descriptors that are conformant to the [NVM Express Boot Specification 1.0](https://nvmexpress.org/specification/nvme-boot-specification/) and it’s interface with the [NVMe-oF messaging type](https://uefi.org/specs/UEFI/2.10/10_Protocols_Device_Path_Protocol.html#nvme-over-fabric-nvme-of-namespace-device-path) to [UEFI System Specification 2.10](https://uefi.org/specs/UEFI/2.10/).
+ 
+Further implementation details relative to the UEFI NvmeOfDxe driver can be found in the [POC Readme](https://github.com/timberland-sig/edk2/blob/timberland_1.0_final/NetworkPkg/NvmeOfDxe/readme.md).
+ 
+## Known Open Issues and Errata
+ 
+- [ ] IPv6 still has general config and interop issues; primarily:
+    - [ ] IPv6 DHCP6 Type 3/4 missing
+    - [ ] IPv6 RA may still be ongoing during connection attempts
+    - [ ] IPv6 SLAAC addresses may not be populated right
+    - [ ] DHCP6_OPT_BOOT_FILE_URL broken/poorly tested, needs better string parsing handling
+- [ ] Hii Currently Missing
+- [ ] OVMF/QEMU: Live Network changes can cause hangs in NVMe-oF driver
```

Comment:
Do we have an issue for this?

I'm not sure if I understand correctly, but is the FW expected at all to handle "live network changes"? I had this discussion with Hannes recently — can EFI handle e.g. a successful NVMe connection which then breaks down before the boot files (bootloader, kernel, initrd etc) are fully downloaded? Hannes and I thought that this just can't be expected to work.

Reply:
> Do we have an issue for this?

yes, https://github.com/timberland-sig/edk2-private/issues/76 in the old repo.  I've still not migrated the open issues from there to here yet. On my list. 

>  but is the FW expected at all to handle "live network changes"

Needs to be clarified in the issue - but the issue was that it causes the driver to hang indefinitely needing a qemu reset.  It should have just errored.

Reply:
> yes, [timberland-sig/edk2-private#76](https://github.com/timberland-sig/edk2-private/issues/76) in the old repo. 

What I reported back then was not a "live" change. It was rather a change applied between reboots, such that the NV data wouldn't match the actual network any more (for example, added or removed vlan tag).

Reply:
ok, i'll strike this for now.  We're going to find new issues anyways so we can always note errata as they come up or are resolved. 
```suggestion
```